turbopack/crates/turbopack-static/src/lib.rs

```diff
@@ -133,21 +132,8 @@ impl ChunkItem for ModuleChunkItem {
     }
 
     #[turbo_tasks::function]
-    async fn references(&self) -> Result<Vc<ModuleReferences>> {
-        Ok(Vc::cell(vec![ResolvedVc::upcast(
-            SingleOutputAssetReference::new(
-                *ResolvedVc::upcast(self.static_asset),
-                Vc::cell(
-                    format!(
-                        "static(url) {}",
-                        self.static_asset.ident().to_string().await?
-                    )
-                    .into(),
-                ),
-            )
-            .to_resolved()
-            .await?,
-        )]))
+    async fn references(&self) -> Result<Vc<OutputAssets>> {
+        Ok(OutputAssets::new(vec![Vc::upcast(*self.static_asset)]))
```

Comment:
```suggestion
    fn references(&self) -> Vc<OutputAssets> {
        Vc::cell(vec![Vc::upcast(*self.static_asset)])
```

There is a difference between `OutputAssets::new` and `Vc::<OutputAssets>::cell` regarding caching. In general you want to use `Vc::cell`, to have this "bubbling invalidation" behavior. `OutputAssets::new` will get you in a world where you have a separate cache entry per list of output assets, which isn't good here